src/vmware-template.xml

```diff
@@ -87,9 +87,9 @@
       <Vendor>{vsphere_product_vendor_name}</Vendor>
       <Version>{vsphere_product_version}</Version>
       <Property ovf:userConfigurable="true" ovf:type="string"
-                ovf:key="guestinfo.coreos.config.data" ovf:value="">
-        <Label>CoreOS config data</Label>
-        <Description>Inline cloud-config data</Description>
+                ovf:key="guestinfo.ignition.config.data" ovf:value="">
```

Comment:
There should also be a [`guestinfo.ignition.config.data.encoding` property](https://github.com/coreos/scripts/blob/9ca5893ba7daef6c78d2618ca2b46c2c7403ab35/build_library/template_vmware.ovf#L36-L40).

Reply:
Added!